getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
 
 ### Exercises:
 
-* create or fork a couple of GitHub repositories for your GitHub account
+* Create or fork a couple of GitHub repositories, and verify that they show in your account as copies. Copies are indicated by the username that proceeds the repository name in all its titles; for example, a repo called *OringinalAuthor/TestRepo* would be renamed to *YourUserName/TestRepo* once you've forked it.
 
 {{% sub_chapter_header "Working with paginated Data" "github-paginated-data" %}}
 
-Often you will request a list of repositories when using GitHub's GraphQL API. Thus it is valuable to have more than one repository listed in your GitHub account, because later you may work with a mechanism called pagination. Pagination was invented to work with large lists of items. Imagine you would have more than 100 repositories in your GitHub account, but your UI only shows 10 of these repositories. It would be a bad idea to transfer the whole list across the wire when performing a network request because only a subset of it is needed. That's where pagination comes into play. Different pagination mechanisms allow you to request only a subset (page) of your list. For instance, these can be the first 10 of your 100 repositories. Later, if you are using pagination with GitHub's GraphQL API, make sure that you adjust the numbers to your own needs. If you have 3 repositories in your GitHub account, it makes sense to request only 2 repositories as your first page, and then again 2 more repositories (which would lead only to 1 more repository, because you haven't any more repositories available) as your second page. Instead, if you would request all 3 repositories as your first page, you wouldn't have any repositories left to request another page of repositories. Then it becomes difficult to see the pagination feature in action. So make sure to adjust the numbers (e.g. limit, offset) to your personal requirements (e.g. available repositories of your GitHub account or available repositories of a GitHub organization).
+ GitHub's GraphQL API allows you to request multiple repositories at once, which is useful for pagination. Pagination is a programming mechanic invented to work with large lists of items. For example, imagine you have more than a hundred repositories in your GitHub account, but your UI only only shows ten of them. Transferring the whole list across the wire for each request is impractical and inefficient, because only a subset is needed at a time, which pagination allows.
+
+Using pagination with GitHub's GraphQL API lets you adjust the numbers to your own needs, so make sure to adjust the numbers (e.g. limit, offset) to your personal requirements (e.g. available repositories of your GitHub account or available repositories of a GitHub organization).  You at least want to have enough repositories in your collection to see the pagination feature in action, so we recommend more than twenty (20), assuming each page will display ten (10).
```

Comment:
... so we recommend more than twenty (20), assuming each page will display ten (10). But you can also go with 5 repositories when displaying only 2.

Reply:
Changed to "or use five(5) repositories when displaying two (2.)"  My opinion: the latter part is an extra example where the point was already made, that pagination is a way to show X amount of results from a larger whole.  Also, an FYI: it is incorrect grammar to start a sentence with the word "but" (same with "so", "hence", "therefore").  These are prepositions and are generally used after commas or semicolons to bridge two clauses together.  Exampe:
* INCORRECT:  I had a bad day.  But tomorrow will be better.
* CORRECT: I had a bad day, but tomorrow will be better.

Reply:
Makes sense! Thanks for clarification.

I just wanted to have a lower number here, because otherwise people may think they have to create 10 or 20 repositories to get started :)